e2etest/pkg/metallb/metallb.go

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier:Apache-2.0
+
+package metallb
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	clientset "k8s.io/client-go/kubernetes"
+)
+
+// SpeakerPods returns the set of pods running the speakers.
```

Comment:
Can we have a func for the ControllerPod as well?

Reply:
Will do